### PR TITLE
[Mellanox] Fix for SN4700 sensors

### PR DESCRIPTION
**Pull request by shlomibitton**
Summary:
Fixes sensor data for 4700
### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Documentation

### CODE REVIEW

File: ansible/group_vars/sonic/sku-sensors-data.yml [hunk 1]
```diff
@@ -1536,87 +1536,127 @@ sensors_checks:
...
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 1.8V T6_7 Rail Curr (in)/curr2_max_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 0.85V VCORE_T6_7 Rail Curr (out)/curr3_max_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 0.85V VCORE_T6_7 Rail Curr (out)/curr3_crit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 1.8V T6_7 Rail Curr (out)/curr4_max_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 1.8V T6_7 Rail Curr (out)/curr4_crit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in1_lcrit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in1_crit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in2_lcrit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in2_crit_alarm
+      - xdpe12284-i2c-5-62/PMIC-1 ASIC 0.85V VCORE_T6_7 Rail (out)/in3_crit_alarm
```
Comment (stephenxs) [inline ansible/group_vars/sonic/sku-sensors-data.yml:1549]: We can't have the same label for both in1 and in2. Doing so will fail the sensor output parser.
I suggest changing to:
```
      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in1)/in1_lcrit_alarm
      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in1)/in1_crit_alarm
      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in2)/in2_lcrit_alarm
      - xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in2)/in2_crit_alarm
```
BTW, does the test pass with this configure? Does it complain something like the following sensor unreadble?
```
xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in1_lcrit_alarm
xdpe12284-i2c-5-62/PMIC-1 PSU 12V Rail (in)/in1_crit_alarm
```

File: ansible/group_vars/sonic/sku-sensors-data.yml [hunk 2]
```diff
@@ -1536,87 +1536,127 @@ sensors_checks:
...
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 1.8V T6_7 Rail Curr (in)/curr2_max_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 0.85V VCORE_T6_7 Rail Curr (out)/curr3_max_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 0.85V VCORE_T6_7 Rail Curr (out)/curr3_crit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 1.8V T6_7 Rail Curr (out)/curr4_max_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 1.8V T6_7 Rail Curr (out)/curr4_crit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 PSU 12V Rail (in)/in1_lcrit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 PSU 12V Rail (in)/in1_crit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 PSU 12V Rail (in)/in2_lcrit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 PSU 12V Rail (in)/in2_crit_alarm
+      - xdpe12284-i2c-5-64/PMIC-2 ASIC 0.85V VCORE_T6_7 Rail (out)/in3_crit_alarm
```
Comment (stephenxs) [inline ansible/group_vars/sonic/sku-sensors-data.yml:1564]: Here the same and there are more in the following.
Can you go over and update them?
Thanks.

File: ansible/group_vars/sonic/sku-sensors-data.yml (hunk 1 above)
Comment (shlomibitton) [reply ansible/group_vars/sonic/sku-sensors-data.yml:1549]: changed, and no all reading are working.

File: ansible/group_vars/sonic/sku-sensors-data.yml (hunk 2 above)
Comment (shlomibitton) [reply ansible/group_vars/sonic/sku-sensors-data.yml:1564]: changed.

**Pull request closed without merging**